server/src/services/auth.service.ts

```diff
@@ -335,6 +337,10 @@ export class AuthService {
       await this.organizationUsersService.create(user, organization, false, manager);
       return this.generateLoginResultPayload(user, organization, false, true, manager);
     });
+
+    await this.metadataService.finishOnboarding(name, email, companyName, companySize, role);
```

Comment:
what if this url `https://hub.tooljet.io/subscribe` is not reachable from the instance? should add a try catch for finish onboarding function
https://github.com/ToolJet/ToolJet/pull/5205/files#diff-61222a9e424956861d68b088bbb08a464f1fa5d8a8c8fa7a3cfa116ae1f8170fR42

Reply:
and can call this function asynchronously